pkgs/development/tools/misc/replacement/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+with python3Packages;
```

Comment:
That's considered a bad habit to put this into such a large scope. See https://nix.dev/anti-patterns/index.html . Remove this and follow the other suggestions.

```suggestion
```